hypervisor/src/kvm/mod.rs

```diff
@@ -330,6 +341,16 @@ impl vm::Vm for KvmVm {
     fn set_state(&self, _state: &VmState) -> vm::Result<()> {
         Ok(())
     }
+
+    // set vmmops interface
+    fn set_vmmops(&mut self, vmmops: Arc<Mutex<dyn VmmOps>>) -> vm::Result<()>{
+        self.vmmops = Some(vmmops);
+        Ok(())
+    }
+    // get vmmops interface
+    fn get_vmmops(&self) -> vm::Result<Arc<Mutex<dyn VmmOps>>>{
+        Ok(self.vmmops.clone().unwrap())
```

Comment:
Please don't `unwrap` unconditionally. This function returns an `Result` so we should be able to pass the error up.